### ISSUE TITLE
Runtime Error on this sample file

### ISSUE THREAD

**Issue by sampad1370** [ccerhan/LibSVMsharp#23]
Hi to all
when i run bSVMsharp.Examples.Classification sample with two blew file,I get runtime Error.
[0066.txt](https://github.com/ccerhan/LibSVMsharp/files/1573188/0066.txt)
[0059.txt](https://github.com/ccerhan/LibSVMsharp/files/1573189/0059.txt)
Use 0066.txt.ini for training set and 0059.txt.ini for test set.
Excuse me for bad english.
![error](https://user-images.githubusercontent.com/7001738/34178066-1ba6d628-e51b-11e7-83ba-29dbceb17ef2.png)

**Comment by qin0** [comment]
I don't know why, but this document needs to be in LF format.
You can try copying your data into the Txt of the example.
